<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width" />
    <title>Eval Output</title>
    <style>
      body {
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica, Arial,
          sans-serif;
      }
      table,
      th,
      td {
        border: 1px solid black;
        border-collapse: collapse;
        text-align: left;
        word-break: break-all;
      }
      th,
      td {
        padding: 5px;
        min-width: 200px;
        white-space: pre-line;
        vertical-align: top;
      }

      tr > td[data-content^='[PASS]'] {
        color: green;
      }
      tr > td[data-content^='[FAIL]'] {
        color: #ad0000;
      }
    </style>
  </head>
  <body>
    <table>
      <thead>
        
        <th>content</th>
        
        <th>keywords</th>
        
        <th>test_description</th>
        
        <th>title</th>
        
        <th>/home/miltondp/projects/others/manubot/manubot-ai-editor-evals/methods/prompts/baseline.txt</th>
        
        <th>/home/miltondp/projects/others/manubot/manubot-ai-editor-evals/methods/prompts/candidate_with_metadata.txt</th>
        
        <th>/home/miltondp/projects/others/manubot/manubot-ai-editor-evals/methods/prompts/candidate.txt</th>
        
      </thead>
      <tbody>
        
        <tr>
          
          <td data-content="Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene&#39;s predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.">Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene's predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves inline math">Preserves inline math</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.41)

Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}
---
Here, we provide a brief overview of the TWAS methods necessary to elucidate our regression framework in subsequent sections (refer to the cited articles for more detailed information). In our notation, $&#92;mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for ease of interpretation (eliminating the need for an intercept term). The variable $&#92;tilde{&#92;mathbf{t}}_l$ denotes the predicted gene expression for all individuals in tissue $l$, calculated as the sum of the product of weights $w_{a}^{l}$ and genotypes $X_{a}$ for SNPs $a$ included in model $l$. Here, $X_a$ represents the genotype of SNP $a$ and $w_{a}$ signifies its weight in tissue prediction model $l$. Additionally, $&#92;mathbf{t}_l$ refers to the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}">[FAIL] (0.41)

Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}
---
Here, we provide a brief overview of the TWAS methods necessary to elucidate our regression framework in subsequent sections (refer to the cited articles for more detailed information). In our notation, $\mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for ease of interpretation (eliminating the need for an intercept term). The variable $\tilde{\mathbf{t}}_l$ denotes the predicted gene expression for all individuals in tissue $l$, calculated as the sum of the product of weights $w_{a}^{l}$ and genotypes $X_{a}$ for SNPs $a$ included in model $l$. Here, $X_a$ represents the genotype of SNP $a$ and $w_{a}$ signifies its weight in tissue prediction model $l$. Additionally, $\mathbf{t}_l$ refers to the standardized version of $\tilde{\mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}</td>
          
          <td data-content="[FAIL] (0.41)

Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods necessary for understanding our regression framework (see the referenced articles for more details). In the following, $&#92;mathbf{y}$ denotes a vector of traits for $n$ individuals, which has been centered for convenience (eliminating the need for an intercept). The predicted expression of a gene in tissue $l$, denoted by $&#92;tilde{&#92;mathbf{t}}_l$, is calculated as the sum of the product of weights $w_{a}^{l}$ and genotypes $X_{a}$ for all SNPs $a$ in the model $&#92;mathrm{model}_l$. Here, $X_a$ represents the genotype of SNP $a$, $w_{a}$ is the weight of SNP $a$ in the tissue prediction model $l$, and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}">[FAIL] (0.41)

Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods necessary for understanding our regression framework (see the referenced articles for more details). In the following, $\mathbf{y}$ denotes a vector of traits for $n$ individuals, which has been centered for convenience (eliminating the need for an intercept). The predicted expression of a gene in tissue $l$, denoted by $\tilde{\mathbf{t}}_l$, is calculated as the sum of the product of weights $w_{a}^{l}$ and genotypes $X_{a}$ for all SNPs $a$ in the model $\mathrm{model}_l$. Here, $X_a$ represents the genotype of SNP $a$, $w_{a}$ is the weight of SNP $a$ in the tissue prediction model $l$, and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}</td>
          
          <td data-content="[FAIL] (0.41)

Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods necessary to understand our regression framework later on. For more in-depth information, please refer to the relevant articles cited. In the context of our study, $&#92;mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for ease of interpretation (eliminating the need for an intercept term). The variable $&#92;tilde{&#92;mathbf{t}}_l$ denotes the predicted gene expression for all individuals in tissue $l$, calculated as the sum of the product of weights $w_{a}^{l}$ and genotypes $X_{a}$ for SNPs $a$ included in the model for tissue $l$. Here, $w_{a}$ represents the weight assigned to SNP $a$ in tissue prediction model $l$, and $&#92;mathbf{t}_l$ is the standardized form of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}">[FAIL] (0.41)

Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods necessary to understand our regression framework later on. For more in-depth information, please refer to the relevant articles cited. In the context of our study, $\mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for ease of interpretation (eliminating the need for an intercept term). The variable $\tilde{\mathbf{t}}_l$ denotes the predicted gene expression for all individuals in tissue $l$, calculated as the sum of the product of weights $w_{a}^{l}$ and genotypes $X_{a}$ for SNPs $a$ included in the model for tissue $l$. Here, $w_{a}$ represents the weight assigned to SNP $a$ in tissue prediction model $l$, and $\mathbf{t}_l$ is the standardized form of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}</td>
          
        </tr>
        
        <tr>
          
          <td data-content="Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene&#39;s predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.">Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene's predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves inline math">Preserves inline math</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.41)

Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}
---
In this section, we outline the TWAS methods essential for understanding our regression framework (refer to the cited articles for more detailed information). Throughout the text, $&#92;mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for ease of analysis (eliminating the need for an intercept). The predicted gene expression for all individuals in tissue $l$, denoted as $&#92;tilde{&#92;mathbf{t}}_l$, is calculated as the sum of the weights $w_{a}^{l}$ multiplied by the genotype $X_{a}$ for each SNP $a$ in the model $&#92;mathrm{model}_l$. Here, $w_{a}$ signifies the weight of SNP $a$ in the tissue prediction model $l$, and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}">[FAIL] (0.41)

Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}
---
In this section, we outline the TWAS methods essential for understanding our regression framework (refer to the cited articles for more detailed information). Throughout the text, $\mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for ease of analysis (eliminating the need for an intercept). The predicted gene expression for all individuals in tissue $l$, denoted as $\tilde{\mathbf{t}}_l$, is calculated as the sum of the weights $w_{a}^{l}$ multiplied by the genotype $X_{a}$ for each SNP $a$ in the model $\mathrm{model}_l$. Here, $w_{a}$ signifies the weight of SNP $a$ in the tissue prediction model $l$, and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}</td>
          
          <td data-content="[PASS] (0.47)

Here we provide a brief overview of the TWAS methods essential for understanding our regression framework in subsequent sections (for more detailed information, refer to the cited articles). In our notation, $&#92;mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for convenience to eliminate the need for an intercept term. The predicted gene expression for all individuals in tissue $l$, denoted as $&#92;tilde{&#92;mathbf{t}}_l$, is calculated as $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$, where $X_a$ represents the genotype of SNP $a$ and $w_{a}$ is the weight of SNP $a$ in tissue prediction model $l$. Additionally, $&#92;mathbf{t}_l$ refers to the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$, with a mean of zero and standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we provide a brief overview of the TWAS methods essential for understanding our regression framework in subsequent sections (for more detailed information, refer to the cited articles). In our notation, $\mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for convenience to eliminate the need for an intercept term. The predicted gene expression for all individuals in tissue $l$, denoted as $\tilde{\mathbf{t}}_l$, is calculated as $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$, where $X_a$ represents the genotype of SNP $a$ and $w_{a}$ is the weight of SNP $a$ in tissue prediction model $l$. Additionally, $\mathbf{t}_l$ refers to the standardized version of $\tilde{\mathbf{t}}_l$, with a mean of zero and standard deviation of one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[FAIL] (0.41)

Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods essential for understanding our regression framework in subsequent sections. In this context, $&#92;mathbf{y}$ denotes a vector representing traits for $n$ individuals, which has been centered for convenience to eliminate the need for an intercept. The variable $&#92;tilde{&#92;mathbf{t}}_l$ represents the predicted gene expression of all individuals in tissue $l$, calculated as the sum of weights $w_{a}^{l}$ multiplied by the genotype $X_{a}$ for SNP $a$ included in the model $&#92;mathrm{model}_l$. Here, $w_{a}$ signifies the weight of SNP $a$ in the tissue prediction model $l$. Furthermore, $&#92;mathbf{t}_l$ refers to the standardized form of $&#92;tilde{&#92;mathbf{t}}_l$, with a mean of zero and a standard deviation of one. For more detailed information, readers are encouraged to consult the relevant articles referenced in this manuscript.

Fail Reason: Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}">[FAIL] (0.41)

Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods essential for understanding our regression framework in subsequent sections. In this context, $\mathbf{y}$ denotes a vector representing traits for $n$ individuals, which has been centered for convenience to eliminate the need for an intercept. The variable $\tilde{\mathbf{t}}_l$ represents the predicted gene expression of all individuals in tissue $l$, calculated as the sum of weights $w_{a}^{l}$ multiplied by the genotype $X_{a}$ for SNP $a$ included in the model $\mathrm{model}_l$. Here, $w_{a}$ signifies the weight of SNP $a$ in the tissue prediction model $l$. Furthermore, $\mathbf{t}_l$ refers to the standardized form of $\tilde{\mathbf{t}}_l$, with a mean of zero and a standard deviation of one. For more detailed information, readers are encouraged to consult the relevant articles referenced in this manuscript.

Fail Reason: Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}</td>
          
        </tr>
        
        <tr>
          
          <td data-content="Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene&#39;s predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.">Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene's predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves inline math">Preserves inline math</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.41)

Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}
---
In this section, we provide a brief overview of the TWAS methods necessary for understanding our regression framework in subsequent sections (refer to the cited articles for further details). Throughout the following discussion, $&#92;mathbf{y}$ represents a vector of traits for $n$ individuals that has been centered for ease of interpretation, eliminating the need for an intercept. The predicted expression of a gene for all individuals in tissue $l$, denoted as $&#92;tilde{&#92;mathbf{t}}_l$, is calculated as the sum of the products of the weights $w_{a}^{l}$ and genotypes $X_{a}$ for each SNP $a$ in the model $&#92;mathrm{model}_l$. Here, $X_a$ represents the genotype of SNP $a$ and $w_{a}$ signifies its weight in the tissue prediction model $l$. Additionally, $&#92;mathbf{t}_l$ refers to the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}">[FAIL] (0.41)

Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}
---
In this section, we provide a brief overview of the TWAS methods necessary for understanding our regression framework in subsequent sections (refer to the cited articles for further details). Throughout the following discussion, $\mathbf{y}$ represents a vector of traits for $n$ individuals that has been centered for ease of interpretation, eliminating the need for an intercept. The predicted expression of a gene for all individuals in tissue $l$, denoted as $\tilde{\mathbf{t}}_l$, is calculated as the sum of the products of the weights $w_{a}^{l}$ and genotypes $X_{a}$ for each SNP $a$ in the model $\mathrm{model}_l$. Here, $X_a$ represents the genotype of SNP $a$ and $w_{a}$ signifies its weight in the tissue prediction model $l$. Additionally, $\mathbf{t}_l$ refers to the standardized version of $\tilde{\mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}</td>
          
          <td data-content="[PASS] (0.47)

Here we provide a brief overview of the TWAS methods essential for understanding our regression framework (for more details, refer to the cited articles). Throughout this discussion, $&#92;mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for ease of interpretation (eliminating the need for an intercept). The expression $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ denotes the predicted gene expression of the gene in tissue $l$ for all individuals, where $X_a$ corresponds to the genotype of SNP $a$ and $w_{a}$ is the weight of SNP $a$ in tissue prediction model $l$. Additionally, $&#92;mathbf{t}_l$ denotes the standardized form of $&#92;tilde{&#92;mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we provide a brief overview of the TWAS methods essential for understanding our regression framework (for more details, refer to the cited articles). Throughout this discussion, $\mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for ease of interpretation (eliminating the need for an intercept). The expression $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ denotes the predicted gene expression of the gene in tissue $l$ for all individuals, where $X_a$ corresponds to the genotype of SNP $a$ and $w_{a}$ is the weight of SNP $a$ in tissue prediction model $l$. Additionally, $\mathbf{t}_l$ denotes the standardized form of $\tilde{\mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[FAIL] (0.41)

Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods essential for understanding our regression framework in subsequent sections. For further details, readers are encouraged to consult the relevant literature. In this context, $&#92;mathbf{y}$ denotes a vector representing traits for $n$ individuals, which has been centered for convenience to eliminate the need for an intercept. The variable $&#92;tilde{&#92;mathbf{t}}_l$ represents the predicted gene expression for all individuals in tissue $l$, calculated as the sum of the products of weights $w_{a}^{l}$ and genotypes $X_{a}$ for SNPs $a$ included in model $l$. Here, $w_{a}^{l}$ denotes the weight of SNP $a$ in the tissue prediction model $l$. Additionally, $&#92;mathbf{t}_l$ refers to the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}">[FAIL] (0.41)

Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods essential for understanding our regression framework in subsequent sections. For further details, readers are encouraged to consult the relevant literature. In this context, $\mathbf{y}$ denotes a vector representing traits for $n$ individuals, which has been centered for convenience to eliminate the need for an intercept. The variable $\tilde{\mathbf{t}}_l$ represents the predicted gene expression for all individuals in tissue $l$, calculated as the sum of the products of weights $w_{a}^{l}$ and genotypes $X_{a}$ for SNPs $a$ included in model $l$. Here, $w_{a}^{l}$ denotes the weight of SNP $a$ in the tissue prediction model $l$. Additionally, $\mathbf{t}_l$ refers to the standardized version of $\tilde{\mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}</td>
          
        </tr>
        
        <tr>
          
          <td data-content="Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene&#39;s predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.">Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene's predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves inline math">Preserves inline math</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.47)

Here we provide a brief overview of the TWAS methods essential for understanding our regression framework (refer to the cited articles for more detailed information). In this study, $&#92;mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for convenience to eliminate the need for an intercept. The predicted expression of a gene for all individuals in tissue $l$ is denoted as $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$, where $X_a$ is the genotype of SNP $a$ and $w_{a}$ is its weight in the tissue prediction model $l$. Additionally, $&#92;mathbf{t}_l$ refers to the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we provide a brief overview of the TWAS methods essential for understanding our regression framework (refer to the cited articles for more detailed information). In this study, $\mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for convenience to eliminate the need for an intercept. The predicted expression of a gene for all individuals in tissue $l$ is denoted as $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$, where $X_a$ is the genotype of SNP $a$ and $w_{a}$ is its weight in the tissue prediction model $l$. Additionally, $\mathbf{t}_l$ refers to the standardized version of $\tilde{\mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[FAIL] (0.41)

Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods essential for understanding our regression framework (for more details, refer to the cited articles). In our notation, $&#92;mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for convenience (eliminating the need for an intercept). The predicted gene expression for all individuals in tissue $l$, denoted as $&#92;tilde{&#92;mathbf{t}}_l$, is calculated as the sum of the products of weights $w_{a}^{l}$ and genotypes $X_{a}$ for SNPs $a$ in the model $&#92;mathrm{model}_l$. Here, $w_{a}$ represents the weight of SNP $a$ in the tissue prediction model $l$, and $X_a$ is the genotype of the SNP. Additionally, $&#92;mathbf{t}_l$ denotes the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}">[FAIL] (0.41)

Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods essential for understanding our regression framework (for more details, refer to the cited articles). In our notation, $\mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for convenience (eliminating the need for an intercept). The predicted gene expression for all individuals in tissue $l$, denoted as $\tilde{\mathbf{t}}_l$, is calculated as the sum of the products of weights $w_{a}^{l}$ and genotypes $X_{a}$ for SNPs $a$ in the model $\mathrm{model}_l$. Here, $w_{a}$ represents the weight of SNP $a$ in the tissue prediction model $l$, and $X_a$ is the genotype of the SNP. Additionally, $\mathbf{t}_l$ denotes the standardized version of $\tilde{\mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}</td>
          
          <td data-content="[FAIL] (0.41)

Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods essential for understanding our regression framework in subsequent sections. The vector of traits for $n$ individuals, denoted as $&#92;mathbf{y}$, is centered for convenience, eliminating the need for an intercept. The predicted gene expression $&#92;tilde{&#92;mathbf{t}}_l$ in tissue $l$ for all individuals is calculated as the sum of the products of weights $w_{a}^{l}$ and genotypes $X_{a}$ corresponding to SNP $a$ in the model for tissue $l$. The weight $w_{a}$ represents the importance of SNP $a$ in the tissue prediction model $l$. The standardized version of $&#92;tilde{&#92;mathbf{t}}_l$, denoted as $&#92;mathbf{t}_l$, has a mean of zero and a standard deviation of one. For more detailed information, please refer to the relevant articles cited in the references section.

Fail Reason: Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}">[FAIL] (0.41)

Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods essential for understanding our regression framework in subsequent sections. The vector of traits for $n$ individuals, denoted as $\mathbf{y}$, is centered for convenience, eliminating the need for an intercept. The predicted gene expression $\tilde{\mathbf{t}}_l$ in tissue $l$ for all individuals is calculated as the sum of the products of weights $w_{a}^{l}$ and genotypes $X_{a}$ corresponding to SNP $a$ in the model for tissue $l$. The weight $w_{a}$ represents the importance of SNP $a$ in the tissue prediction model $l$. The standardized version of $\tilde{\mathbf{t}}_l$, denoted as $\mathbf{t}_l$, has a mean of zero and a standard deviation of one. For more detailed information, please refer to the relevant articles cited in the references section.

Fail Reason: Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}</td>
          
        </tr>
        
        <tr>
          
          <td data-content="Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene&#39;s predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.">Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene's predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves inline math">Preserves inline math</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.41)

Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}
---
In this section, we provide a brief overview of the TWAS methods that will be used to explain our regression framework later on. For more detailed information, please refer to the cited articles. Throughout the following discussion, we will denote $&#92;mathbf{y}$ as a vector representing traits for $n$ individuals, which has been centered for convenience (eliminating the need for an intercept). The variable $&#92;tilde{&#92;mathbf{t}}_l$ represents the predicted gene expression for all individuals in tissue $l$, calculated as the sum of the weights $w_{a}^{l}$ multiplied by the genotype $X_{a}$ for SNP $a$ in the model for tissue $l$. Additionally, $&#92;mathbf{t}_l$ refers to the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}">[FAIL] (0.41)

Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}
---
In this section, we provide a brief overview of the TWAS methods that will be used to explain our regression framework later on. For more detailed information, please refer to the cited articles. Throughout the following discussion, we will denote $\mathbf{y}$ as a vector representing traits for $n$ individuals, which has been centered for convenience (eliminating the need for an intercept). The variable $\tilde{\mathbf{t}}_l$ represents the predicted gene expression for all individuals in tissue $l$, calculated as the sum of the weights $w_{a}^{l}$ multiplied by the genotype $X_{a}$ for SNP $a$ in the model for tissue $l$. Additionally, $\mathbf{t}_l$ refers to the standardized version of $\tilde{\mathbf{t}}_l$, with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}</td>
          
          <td data-content="[FAIL] (0.41)

Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods essential for understanding our regression framework (refer to the cited articles for more detailed explanations). In the context of our study, $&#92;mathbf{y}$ represents a vector of traits for $n$ individuals that has been centered for convenience, eliminating the need for an intercept. The predicted expression of a gene in tissue $l$, denoted as $&#92;tilde{&#92;mathbf{t}}_l$, is calculated as the sum of the products of the weights $w_{a}^{l}$ and genotypes $X_{a}$ for all SNPs $a$ in the model $&#92;mathrm{model}_l$. Here, $X_a$ represents the genotype of SNP $a$, $w_{a}$ denotes its weight in the tissue-specific prediction model $l$, and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - &#92;tilde{&#92;mathbf{t}}_l = 
 - &#92;sum_{a &#92;in &#92;mathrm{model}_l}">[FAIL] (0.41)

Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}
---
Here we provide a brief overview of the TWAS methods essential for understanding our regression framework (refer to the cited articles for more detailed explanations). In the context of our study, $\mathbf{y}$ represents a vector of traits for $n$ individuals that has been centered for convenience, eliminating the need for an intercept. The predicted expression of a gene in tissue $l$, denoted as $\tilde{\mathbf{t}}_l$, is calculated as the sum of the products of the weights $w_{a}^{l}$ and genotypes $X_{a}$ for all SNPs $a$ in the model $\mathrm{model}_l$. Here, $X_a$ represents the genotype of SNP $a$, $w_{a}$ denotes its weight in the tissue-specific prediction model $l$, and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Fail Reason: Failed to find the following keywords:
 - \tilde{\mathbf{t}}_l = 
 - \sum_{a \in \mathrm{model}_l}</td>
          
          <td data-content="[PASS] (0.47)

Here we provide a brief description of the TWAS methods essential for understanding our regression framework in subsequent sections. In this context, $&#92;mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for convenience to eliminate the need for an intercept. The predicted gene expression $&#92;tilde{&#92;mathbf{t}}_l$ for all individuals in tissue $l$ is calculated as $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$, where $X_a$ denotes the genotype of SNP $a$ and $w_{a}$ is the weight of SNP $a$ in tissue prediction model $l$. Additionally, $&#92;mathbf{t}_l$ represents the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$, with a mean of zero and a standard deviation of one. For more detailed information, please refer to the relevant articles cited in the references section.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we provide a brief description of the TWAS methods essential for understanding our regression framework in subsequent sections. In this context, $\mathbf{y}$ represents a vector of traits for $n$ individuals, which has been centered for convenience to eliminate the need for an intercept. The predicted gene expression $\tilde{\mathbf{t}}_l$ for all individuals in tissue $l$ is calculated as $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$, where $X_a$ denotes the genotype of SNP $a$ and $w_{a}$ is the weight of SNP $a$ in tissue prediction model $l$. Additionally, $\mathbf{t}_l$ represents the standardized version of $\tilde{\mathbf{t}}_l$, with a mean of zero and a standard deviation of one. For more detailed information, please refer to the relevant articles cited in the references section.

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves numbered equations and references to articles">Preserves numbered equations and references to articles</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.73)

Failed to find the following keywords:
 - $$ {#eq:predixcan}
 - $$ {#eq:spredixcan}
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of gene expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;hat{&#92;gamma}_l$ represents the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. As S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (detailed below).

Fail Reason: Failed to find the following keywords:
 - $$ {#eq:predixcan}
 - $$ {#eq:spredixcan}">[FAIL] (0.73)

Failed to find the following keywords:
 - $$ {#eq:predixcan}
 - $$ {#eq:spredixcan}
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of gene expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\hat{\gamma}_l$ represents the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. As S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (detailed below).

Fail Reason: Failed to find the following keywords:
 - $$ {#eq:predixcan}
 - $$ {#eq:spredixcan}</td>
          
          <td data-content="[FAIL] (0.57)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan provides information on the tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression increases or decreases disease risk), we utilized the $z$-scores in our drug repurposing strategy (as detailed below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776">[FAIL] (0.57)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan provides information on the tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression increases or decreases disease risk), we utilized the $z$-scores in our drug repurposing strategy (as detailed below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a simplified version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of gene expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

Here, $&#92;hat{&#92;gamma}_l$ represents the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the equation:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ denotes the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained in the subsequent section).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a simplified version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of gene expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

Here, $\hat{\gamma}_l$ represents the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the equation:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

Here, $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ denotes the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained in the subsequent section).

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves numbered equations and references to articles">Preserves numbered equations and references to articles</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using a univariate model described by the equation:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ represents the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects, indicating whether a higher or lower predicted expression of a gene confers more or less disease risk, we utilized the $z$-scores in our drug repurposing strategy (explained below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using a univariate model described by the equation:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ represents the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

Here, $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects, indicating whether a higher or lower predicted expression of a gene confers more or less disease risk, we utilized the $z$-scores in our drug repurposing strategy (explained below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of gene expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;hat{&#92;gamma}_l$ denotes the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained in detail below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776">[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of gene expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\hat{\gamma}_l$ denotes the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained in detail below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776</td>
          
          <td data-content="[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;hat{&#92;gamma}_l$ denotes the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1">[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\hat{\gamma}_l$ denotes the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves numbered equations and references to articles">Preserves numbered equations and references to articles</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

Here, $&#92;hat{&#92;gamma}_l$ represents the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression correlates with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

Here, $\hat{\gamma}_l$ represents the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

Here, $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression correlates with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of gene expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;hat{&#92;gamma}_l$ denotes the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. Since S-PrediXcan offers tissue-specific direction of effects (e.g., indicating whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (outlined below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1">[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of gene expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\hat{\gamma}_l$ denotes the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. Since S-PrediXcan offers tissue-specific direction of effects (e.g., indicating whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (outlined below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1</td>
          
          <td data-content="[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776">[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves numbered equations and references to articles">Preserves numbered equations and references to articles</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.73)

Failed to find the following keywords:
 - $$ {#eq:predixcan}
 - $$ {#eq:spredixcan}
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of gene expression in a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ 

where $&#92;hat{&#92;gamma}_l$ represents the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the formula

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ 

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (as described below).

Fail Reason: Failed to find the following keywords:
 - $$ {#eq:predixcan}
 - $$ {#eq:spredixcan}">[FAIL] (0.73)

Failed to find the following keywords:
 - $$ {#eq:predixcan}
 - $$ {#eq:spredixcan}
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of gene expression in a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ 

where $\hat{\gamma}_l$ represents the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the formula

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ 

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (as described below).

Fail Reason: Failed to find the following keywords:
 - $$ {#eq:predixcan}
 - $$ {#eq:spredixcan}</td>
          
          <td data-content="[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a condensed version of PrediXcan. PrediXcan represents a trait as a linear function of a gene&#39;s expression in a single tissue using a univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;hat{&#92;gamma}_l$ denotes the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with a variance of $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. PrediXcan necessitates individual-level data for model fitting, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ signifies the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ denotes the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan offers tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression correlates with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy as detailed below.

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776">[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a condensed version of PrediXcan. PrediXcan represents a trait as a linear function of a gene's expression in a single tissue using a univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\hat{\gamma}_l$ denotes the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with a variance of $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. PrediXcan necessitates individual-level data for model fitting, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ signifies the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ denotes the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan offers tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression correlates with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy as detailed below.

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776</td>
          
          <td data-content="[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics through the formula:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (such as whether higher or lower predicted gene expression correlates with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (details provided below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776">[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics through the formula:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (such as whether higher or lower predicted gene expression correlates with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (details provided below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves numbered equations and references to articles">Preserves numbered equations and references to articles</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.73)

Failed to find the following keywords:
 - $$ {#eq:predixcan}
 - $$ {#eq:spredixcan}
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;hat{&#92;gamma}_l$ represents the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing strategy (detailed below).

Fail Reason: Failed to find the following keywords:
 - $$ {#eq:predixcan}
 - $$ {#eq:spredixcan}">[FAIL] (0.73)

Failed to find the following keywords:
 - $$ {#eq:predixcan}
 - $$ {#eq:spredixcan}
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\hat{\gamma}_l$ represents the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing strategy (detailed below).

Fail Reason: Failed to find the following keywords:
 - $$ {#eq:predixcan}
 - $$ {#eq:spredixcan}</td>
          
          <td data-content="[FAIL] (0.57)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan predicts a trait based on the gene&#39;s expression in a single tissue using a linear model represented as:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. The genotype variances and covariances in these TWAS methods are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (indicating if higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776">[FAIL] (0.57)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan predicts a trait based on the gene's expression in a single tissue using a linear model represented as:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. The genotype variances and covariances in these TWAS methods are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (indicating if higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776</td>
          
          <td data-content="[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan offers tissue-specific directions of effects, we utilized the $z$-scores in our drug repurposing strategy (explained below) [1].

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776">[FAIL] (0.53)

Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan offers tissue-specific directions of effects, we utilized the $z$-scores in our drug repurposing strategy (explained below) [1].

Fail Reason: Failed to reference the following articles:
 - @doi:10.1038/ng.3367
 - @doi:10.1038/s41467-018-03621-1
 - @doi:10.1126/science.aaz1776</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where &#92;mathbf{t}_l is the gene predicted expression in tissue $l$, $&#92;beta_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;Sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where \mathbf{t}_l is the gene predicted expression in tissue $l$, $\beta_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\Sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes incorrect math references">Fixes incorrect math references</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.28)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;Sigma}_a$ -&gt; $&#92;hat{&#92;sigma}_a$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan utilizes a univariate model that represents the trait as a linear function of the gene&#39;s expression in a single tissue:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

Here, $&#92;mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;Sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;Sigma}_a$ -&gt; $&#92;hat{&#92;sigma}_a$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.28)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\Sigma}_a$ -> $\hat{\sigma}_a$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan utilizes a univariate model that represents the trait as a linear function of the gene's expression in a single tissue:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

Here, $\mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

Here, $\hat{\Sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\Sigma}_a$ -> $\hat{\sigma}_a$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of gene expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size (or regression coefficient), and $&#92;epsilon_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression increases or decreases disease risk), we utilized the $z$-scores in our drug repurposing strategy (as detailed below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.34)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of gene expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size (or regression coefficient), and $\epsilon_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression increases or decreases disease risk), we utilized the $z$-scores in our drug repurposing strategy (as detailed below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
          <td data-content="[FAIL] (0.41)

Failed to fix the following expressions:
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. PrediXcan requires individual-level data to construct this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;sigma}_a$ denotes the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ represents the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan offers tissue-specific direction of effects, such as indicating whether higher or lower predicted gene expression is associated with increased disease risk, we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.41)

Failed to fix the following expressions:
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. PrediXcan requires individual-level data to construct this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

Here, $\hat{\sigma}_a$ denotes the variance of SNP $a$, $\hat{\sigma}_l$ represents the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan offers tissue-specific direction of effects, such as indicating whether higher or lower predicted gene expression is associated with increased disease risk, we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where &#92;mathbf{t}_l is the gene predicted expression in tissue $l$, $&#92;beta_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;Sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where \mathbf{t}_l is the gene predicted expression in tissue $l$, $\beta_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\Sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes incorrect math references">Fixes incorrect math references</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to construct this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression increases or decreases disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.34)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to construct this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression increases or decreases disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
          <td data-content="[FAIL] (0.41)

Failed to fix the following expressions:
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of gene expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan offers tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained in detail below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.41)

Failed to fix the following expressions:
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of gene expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan offers tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained in detail below).

Fail Reason: Failed to fix the following expressions:
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
          <td data-content="[FAIL] (0.41)

Failed to fix the following expressions:
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan models a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. Since S-PrediXcan provides tissue-specific direction of effects, such as whether higher or lower predicted gene expression is associated with increased or decreased disease risk, we utilized the $z$-scores in our drug repurposing strategy, as detailed below.

Fail Reason: Failed to fix the following expressions:
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.41)

Failed to fix the following expressions:
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan models a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

Here, $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. Since S-PrediXcan provides tissue-specific direction of effects, such as whether higher or lower predicted gene expression is associated with increased or decreased disease risk, we utilized the $z$-scores in our drug repurposing strategy, as detailed below.

Fail Reason: Failed to fix the following expressions:
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where &#92;mathbf{t}_l is the gene predicted expression in tissue $l$, $&#92;beta_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;Sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where \mathbf{t}_l is the gene predicted expression in tissue $l$, $\beta_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\Sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes incorrect math references">Fixes incorrect math references</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.34)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

Here, $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$   (1)

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan approximates PrediXcan $z$-scores using only summary statistics from GWAS with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$   (2)

where $&#92;hat{&#92;sigma}_a$ denotes the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ represents the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression increases or decreases disease risk), we utilized the $z$-scores in our drug repurposing strategy as detailed below.

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.34)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$   (1)

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan approximates PrediXcan $z$-scores using only summary statistics from GWAS with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$   (2)

where $\hat{\sigma}_a$ denotes the variance of SNP $a$, $\hat{\sigma}_l$ represents the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression increases or decreases disease risk), we utilized the $z$-scores in our drug repurposing strategy as detailed below.

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
          <td data-content="[FAIL] (0.28)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;Sigma}_a$ -&gt; $&#92;hat{&#92;sigma}_a$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l, 
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the equation:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)}, 
$$ {#eq:spredixcan}

where $&#92;hat{&#92;Sigma}_a$ denotes the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ represents the variance of predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan offers tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression correlates with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;Sigma}_a$ -&gt; $&#92;hat{&#92;sigma}_a$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.28)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\Sigma}_a$ -> $\hat{\sigma}_a$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l, 
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the equation:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)}, 
$$ {#eq:spredixcan}

where $\hat{\Sigma}_a$ denotes the variance of SNP $a$, $\hat{\sigma}_l$ represents the variance of predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan offers tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression correlates with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\Sigma}_a$ -> $\hat{\sigma}_a$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where &#92;mathbf{t}_l is the gene predicted expression in tissue $l$, $&#92;beta_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;Sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where \mathbf{t}_l is the gene predicted expression in tissue $l$, $\beta_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\Sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes incorrect math references">Fixes incorrect math references</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (as described below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.34)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (as described below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

Here, $&#92;mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. While PrediXcan requires individual-level data for model fitting, S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics, with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

In this equation, $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. The genotype variances and covariances in these Transcriptome-Wide Association Study (TWAS) methods are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. Since S-PrediXcan provides information on the tissue-specific direction of effects, such as whether higher or lower predicted gene expression is associated with increased or decreased disease risk, we utilized the $z$-scores in our drug repurposing strategy, as detailed below.

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.34)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

Here, $\mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. While PrediXcan requires individual-level data for model fitting, S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics, with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

In this equation, $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. The genotype variances and covariances in these Transcriptome-Wide Association Study (TWAS) methods are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. Since S-PrediXcan provides information on the tissue-specific direction of effects, such as whether higher or lower predicted gene expression is associated with increased or decreased disease risk, we utilized the $z$-scores in our drug repurposing strategy, as detailed below.

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan offers tissue-specific direction of effects, such as whether higher or lower predicted gene expression is associated with increased or decreased disease risk, we utilized the $z$-scores in our drug repurposing strategy (explained in detail below) [1].

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.34)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan offers tissue-specific direction of effects, such as whether higher or lower predicted gene expression is associated with increased or decreased disease risk, we utilized the $z$-scores in our drug repurposing strategy (explained in detail below) [1].

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where &#92;mathbf{t}_l is the gene predicted expression in tissue $l$, $&#92;beta_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;Sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where \mathbf{t}_l is the gene predicted expression in tissue $l$, $\beta_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\Sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes incorrect math references">Fixes incorrect math references</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression correlates with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.34)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\mathbf{t}_l$ represents the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression correlates with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
          <td data-content="[FAIL] (0.41)

Failed to fix the following expressions:
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan offers tissue-specific direction of effects (indicating whether higher or lower predicted gene expression contributes to increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (as detailed below) [@doi:10.1038/s41467-018-03621-1; @doi:10.1038/ng.3367; @doi:10.1126/science.aaz1776].

Fail Reason: Failed to fix the following expressions:
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.41)

Failed to fix the following expressions:
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Since S-PrediXcan offers tissue-specific direction of effects (indicating whether higher or lower predicted gene expression contributes to increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (as detailed below) [@doi:10.1038/s41467-018-03621-1; @doi:10.1038/ng.3367; @doi:10.1126/science.aaz1776].

Fail Reason: Failed to fix the following expressions:
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Studies (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. Since S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with more or less disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
 - $&#92;hat{&#92;beta}_l$ -&gt; $&#92;hat{&#92;beta}_a$">[FAIL] (0.34)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

Here, $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Studies (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. Since S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with more or less disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
 - $\hat{\beta}_l$ -> $\hat{\beta}_a$</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes wrong latex expressions">Fixes wrong latex expressions</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l},
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ represents the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Given that S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (as described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l},
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ represents the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

Here, $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Given that S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (as described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.22)

Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ denotes the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ signifies the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ indicates the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan offers tissue-specific direction of effects, such as whether a higher or lower predicted gene expression correlates with increased or decreased disease risk, we utilized the $z$-scores in our drug repurposing strategy as detailed below.

Fail Reason: Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.22)

Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ denotes the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ signifies the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ indicates the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan offers tissue-specific direction of effects, such as whether a higher or lower predicted gene expression correlates with increased or decreased disease risk, we utilized the $z$-scores in our drug repurposing strategy as detailed below.

Fail Reason: Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of the gene&#39;s expression in a single tissue using a univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l} &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ denotes the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following equation:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. As S-PrediXcan offers tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression correlates with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of the gene's expression in a single tissue using a univariate model:

$$
\mathbf{y} = \mathbf{t_l} \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ denotes the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following equation:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

Here, $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. As S-PrediXcan offers tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression correlates with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes wrong latex expressions">Fixes wrong latex expressions</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.22)

Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ represents the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to construct this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. As S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (outlined below).

Fail Reason: Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.22)

Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ represents the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to construct this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. As S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (outlined below).

Fail Reason: Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene&#39;s expression in a single tissue using a univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l} &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ denotes the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the equation:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained in detail below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene's expression in a single tissue using a univariate model:

$$
\mathbf{y} = \mathbf{t_l} \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ denotes the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the equation:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained in detail below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.22)

Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan models a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ represents the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan provides tissue-specific direction of effects, such as whether a higher or lower predicted gene expression is associated with increased or decreased disease risk, we utilized the $z$-scores in our drug repurposing strategy (explained in detail below).

Fail Reason: Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.22)

Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan models a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ represents the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan provides tissue-specific direction of effects, such as whether a higher or lower predicted gene expression is associated with increased or decreased disease risk, we utilized the $z$-scores in our drug repurposing strategy (explained in detail below).

Fail Reason: Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes wrong latex expressions">Fixes wrong latex expressions</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan employs a univariate model that represents a trait as a linear function of a gene&#39;s expression in a single tissue:

$$
&#92;mathbf{y} = &#92;mathbf{t_l} &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ denotes the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to construct this model, whereas S-PrediXcan estimates PrediXcan $z$-scores utilizing only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ denotes the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. As S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers greater disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan employs a univariate model that represents a trait as a linear function of a gene's expression in a single tissue:

$$
\mathbf{y} = \mathbf{t_l} \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ denotes the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to construct this model, whereas S-PrediXcan estimates PrediXcan $z$-scores utilizing only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ denotes the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. As S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers greater disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.22)

Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ denotes the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. PrediXcan requires individual-level data to construct this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ denotes the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan offers tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy, as detailed below.

Fail Reason: Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.22)

Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents the trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ denotes the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. PrediXcan requires individual-level data to construct this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ denotes the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan offers tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy, as detailed below.

Fail Reason: Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan models a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l} &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ represents the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ represents the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ represents the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing strategy (as described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan models a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l} \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ represents the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ represents the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ represents the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing strategy (as described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes wrong latex expressions">Fixes wrong latex expressions</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.22)

Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of gene expression in a single tissue using a univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ represents the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics through the formula:

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. As S-PrediXcan offers tissue-specific directions of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (detailed below).

Fail Reason: Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.22)

Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of gene expression in a single tissue using a univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ represents the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics through the formula:

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. As S-PrediXcan offers tissue-specific directions of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (detailed below).

Fail Reason: Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan predicts a trait based on the gene&#39;s expression in a single tissue using a linear model represented as:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l} + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ denotes the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the equation:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Given that S-PrediXcan offers tissue-specific direction of effects, we utilized the $z$-scores in our drug repurposing strategy, as detailed below.

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan predicts a trait based on the gene's expression in a single tissue using a linear model represented as:

$$
\mathbf{y} = \mathbf{t_l \gamma_l} + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ denotes the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the equation:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. Given that S-PrediXcan offers tissue-specific direction of effects, we utilized the $z$-scores in our drug repurposing strategy, as detailed below.

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma}_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the equation:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ denotes the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. As S-PrediXcan offers tissue-specific direction of effects, such as whether a higher or lower predicted gene expression is associated with increased or decreased disease risk, we utilized the $z$-scores in our drug repurposing strategy (explained in the following section).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma}_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is determined by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the equation:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ denotes the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. As S-PrediXcan offers tissue-specific direction of effects, such as whether a higher or lower predicted gene expression is associated with increased or decreased disease risk, we utilized the $z$-scores in our drug repurposing strategy (explained in the following section).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes wrong latex expressions">Fixes wrong latex expressions</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of gene expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l},
$$

where $&#92;hat{&#92;gamma}_l$ represents the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ denotes the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$

where $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (outlined below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a condensed version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of gene expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l},
$$

where $\hat{\gamma}_l$ represents the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ denotes the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, whereas S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$

where $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing strategy (outlined below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.22)

Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
The approach known as S-PrediXcan is a condensed version of PrediXcan. PrediXcan functions by representing a trait as a linear relationship with a gene&#39;s expression in a single tissue, employing a univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

Here, $&#92;hat{&#92;gamma}_l$ denotes the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan necessitates individual-level data for model fitting, whereas S-PrediXcan estimates PrediXcan $z$-scores solely based on GWAS summary statistics, given by:

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ signifies the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. As S-PrediXcan offers tissue-specific directional effects (e.g., whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained in the subsequent section).

Fail Reason: Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.22)

Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l 
---
The approach known as S-PrediXcan is a condensed version of PrediXcan. PrediXcan functions by representing a trait as a linear relationship with a gene's expression in a single tissue, employing a univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

Here, $\hat{\gamma}_l$ denotes the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan necessitates individual-level data for model fitting, whereas S-PrediXcan estimates PrediXcan $z$-scores solely based on GWAS summary statistics, given by:

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

Here, $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ signifies the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel [@doi:10.1126/science.aaz1776]. As S-PrediXcan offers tissue-specific directional effects (e.g., whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained in the subsequent section).

Fail Reason: Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.22)

Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene&#39;s expression in a single tissue using a univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

Here, $&#92;hat{&#92;gamma}_l$ denotes the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;sigma}_a$ represents the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ denotes the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan offers tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained in the subsequent section).

Fail Reason: Failed to fix the following expressions:
 - &#92;appro  -&gt; &#92;approx 
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.22)

Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan is a simplified version of PrediXcan. PrediXcan represents a trait as a linear function of a gene's expression in a single tissue using a univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

Here, $\hat{\gamma}_l$ denotes the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by calculating the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data for model fitting, while S-PrediXcan estimates PrediXcan $z$-scores using only GWAS summary statistics with the formula:

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

Here, $\hat{\sigma}_a$ represents the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ denotes the estimated effect size of SNP $a$ from the GWAS. In these Transcriptome-Wide Association Study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) as the reference panel. As S-PrediXcan offers tissue-specific direction of effects (indicating whether higher or lower predicted gene expression is associated with increased disease risk), we utilized the $z$-scores in our drug repurposing strategy (explained in the subsequent section).

Fail Reason: Failed to fix the following expressions:
 - \appro  -> \approx 
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
        </tr>
        
      </tbody>
    </table>
  </body>
</html>
