<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width" />
    <title>Eval Output</title>
    <style>
      body {
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica, Arial,
          sans-serif;
      }
      table,
      th,
      td {
        border: 1px solid black;
        border-collapse: collapse;
        text-align: left;
        word-break: break-all;
      }
      th,
      td {
        padding: 5px;
        min-width: 200px;
        white-space: pre-line;
        vertical-align: top;
      }

      tr > td[data-content^='[PASS]'] {
        color: green;
      }
      tr > td[data-content^='[FAIL]'] {
        color: #ad0000;
      }
    </style>
  </head>
  <body>
    <table>
      <thead>
        
        <th>content</th>
        
        <th>keywords</th>
        
        <th>test_description</th>
        
        <th>title</th>
        
        <th>/home/miltondp/projects/others/manubot/manubot-ai-editor-evals/methods/prompts/baseline.txt</th>
        
        <th>/home/miltondp/projects/others/manubot/manubot-ai-editor-evals/methods/prompts/candidate_with_metadata.txt</th>
        
        <th>/home/miltondp/projects/others/manubot/manubot-ai-editor-evals/methods/prompts/candidate.txt</th>
        
      </thead>
      <tbody>
        
        <tr>
          
          <td data-content="Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene&#39;s predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.">Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene's predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves inline math">Preserves inline math</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.47)

Here we briefly provide the details about these transcriptome-wide association study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). Let $&#92;mathbf{y}$ denote a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ represents the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$ and $w_{a}^{l}$ is its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we briefly provide the details about these transcriptome-wide association study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). Let $\mathbf{y}$ denote a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ represents the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$ and $w_{a}^{l}$ is its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.47)

We briefly provide the details about these Transcriptome-Wide Association Study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of traits for $n$ individuals, centered for convenience (so that no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.

Pass Reason: All assertions passed">[PASS] (0.47)

We briefly provide the details about these Transcriptome-Wide Association Study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of traits for $n$ individuals, centered for convenience (so that no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.47)

Here, we briefly provide the necessary details about these Transcriptome-Wide Association Study (TWAS) methods to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a centered vector of traits for $n$ individuals (eliminating the need for an intercept); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ represents the gene&#39;s predicted expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here, we briefly provide the necessary details about these Transcriptome-Wide Association Study (TWAS) methods to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a centered vector of traits for $n$ individuals (eliminating the need for an intercept); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ represents the gene's predicted expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene&#39;s predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.">Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene's predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves inline math">Preserves inline math</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.47)

Here we briefly provide the details about these transcriptome-wide association study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). Let $&#92;mathbf{y}$ denote a vector of traits for $n$ individuals, centered for convenience (so that no intercept is necessary). $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ represents the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}^{l}$ is its weight in the tissue prediction model $l$. $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we briefly provide the details about these transcriptome-wide association study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). Let $\mathbf{y}$ denote a vector of traits for $n$ individuals, centered for convenience (so that no intercept is necessary). $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ represents the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}^{l}$ is its weight in the tissue prediction model $l$. $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.47)

Here we briefly provide the necessary details about these Transcriptome-Wide Association Study (TWAS) methods to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we briefly provide the necessary details about these Transcriptome-Wide Association Study (TWAS) methods to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.47)

In this section, we briefly provide the details about the Transcriptome-Wide Association Study (TWAS) methods necessary to explain our regression framework later (see the referenced articles for more information). We refer to $&#92;mathbf{y}$ as a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ represents the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}^{l}$ is its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

In this section, we briefly provide the details about the Transcriptome-Wide Association Study (TWAS) methods necessary to explain our regression framework later (see the referenced articles for more information). We refer to $\mathbf{y}$ as a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ represents the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}^{l}$ is its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene&#39;s predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.">Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene's predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves inline math">Preserves inline math</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.47)

Here we briefly provide the details about these transcriptome-wide association study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). Let $&#92;mathbf{y}$ denote a vector of traits for $n$ individuals, centered for convenience (so that no intercept is necessary). Let $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ represent the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}^{l}$ is its weight in the tissue prediction model $l$. Furthermore, let $&#92;mathbf{t}_l$ denote the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we briefly provide the details about these transcriptome-wide association study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). Let $\mathbf{y}$ denote a vector of traits for $n$ individuals, centered for convenience (so that no intercept is necessary). Let $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ represent the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}^{l}$ is its weight in the tissue prediction model $l$. Furthermore, let $\mathbf{t}_l$ denote the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.47)

Here we briefly provide the necessary details about these Transcriptome-Wide Association Study (TWAS) methods to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of centered traits for $n$ individuals (for convenience, no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$ and $w_{a}$ is its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we briefly provide the necessary details about these Transcriptome-Wide Association Study (TWAS) methods to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of centered traits for $n$ individuals (for convenience, no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$ and $w_{a}$ is its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.47)

Here, we briefly provide the details about these Transcriptome-Wide Association Study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene&#39;s predicted expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here, we briefly provide the details about these Transcriptome-Wide Association Study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene's predicted expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene&#39;s predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.">Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene's predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves inline math">Preserves inline math</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.47)

Here we briefly provide the details about these transcriptome-wide association study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). Let $&#92;mathbf{y}$ denote a vector of traits for $n$ individuals, centered for convenience (so that no intercept is necessary). Let $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ represent the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}^{l}$ is its weight in the tissue prediction model $l$. Furthermore, let $&#92;mathbf{t}_l$ denote the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we briefly provide the details about these transcriptome-wide association study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). Let $\mathbf{y}$ denote a vector of traits for $n$ individuals, centered for convenience (so that no intercept is necessary). Let $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ represent the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}^{l}$ is its weight in the tissue prediction model $l$. Furthermore, let $\mathbf{t}_l$ denote the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.47)

Here we briefly provide the necessary details about these Transcriptome-Wide Association Study (TWAS) methods to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of centered traits for $n$ individuals (no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we briefly provide the necessary details about these Transcriptome-Wide Association Study (TWAS) methods to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of centered traits for $n$ individuals (no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.47)

In this section, we briefly provide the necessary details about the Transcriptome-Wide Association Study (TWAS) methods to explain our regression framework later (refer to the referenced articles for more comprehensive information). Throughout the text, we denote $&#92;mathbf{y}$ as a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ represents the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

In this section, we briefly provide the necessary details about the Transcriptome-Wide Association Study (TWAS) methods to explain our regression framework later (refer to the referenced articles for more comprehensive information). Throughout the text, we denote $\mathbf{y}$ as a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ represents the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene&#39;s predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.">Here we briefly provide the details about these TWAS methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of traits for $n$ individuals that is centered for convenience (so that no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the gene's predicted expression for all individuals in tissue $l$, $X_a$ is the genotype of SNP $a$ and $w_{a}$ its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with mean equal to zero and standard deviation equal to one.</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves inline math">Preserves inline math</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.47)

Here we briefly provide the details about these transcriptome-wide association study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). Let $&#92;mathbf{y}$ be a vector of centered traits for $n$ individuals (no intercept is necessary); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ represents the gene&#39;s predicted expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$ and $w_{a}$ is its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we briefly provide the details about these transcriptome-wide association study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). Let $\mathbf{y}$ be a vector of centered traits for $n$ individuals (no intercept is necessary); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ represents the gene's predicted expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$ and $w_{a}$ is its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.47)

Here we briefly provide the necessary details about these Transcriptome-Wide Association Study (TWAS) methods to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ is the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here we briefly provide the necessary details about these Transcriptome-Wide Association Study (TWAS) methods to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ is the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}$ is its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.47)

Here, we briefly provide the details about these Transcriptome-Wide Association Study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $&#92;mathbf{y}$ as a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $&#92;tilde{&#92;mathbf{t}}_l = &#92;sum_{a &#92;in &#92;mathrm{model}_l} w_{a}^{l} X_{a}$ represents the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}^{l}$ is its weight in the tissue prediction model $l$; and $&#92;mathbf{t}_l$ is the standardized version of $&#92;tilde{&#92;mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed">[PASS] (0.47)

Here, we briefly provide the details about these Transcriptome-Wide Association Study (TWAS) methods that are necessary to explain our regression framework later (see the referenced articles for more information). In the following, we refer to $\mathbf{y}$ as a vector of centered traits for $n$ individuals (eliminating the need for an intercept); $\tilde{\mathbf{t}}_l = \sum_{a \in \mathrm{model}_l} w_{a}^{l} X_{a}$ represents the predicted gene expression for all individuals in tissue $l$, where $X_a$ is the genotype of single nucleotide polymorphism (SNP) $a$, and $w_{a}^{l}$ is its weight in the tissue prediction model $l$; and $\mathbf{t}_l$ is the standardized version of $\tilde{\mathbf{t}}_l$ with a mean of zero and a standard deviation of one.

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves numbered equations and references to articles">Preserves numbered equations and references to articles</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.66)

Failed to find the following keywords:
 - &#92;hat{z}_{l}
 - &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)
 - &#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)}
 - $&#92;hat{&#92;sigma}_a$
 - $&#92;hat{&#92;sigma}_l$
 - $&#92;hat{&#92;beta}_a$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $z_{l}=&#92;gamma_l / &#92;mathrm{se}(&#92;gamma_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
z_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;sigma_a}{&#92;sigma_l} &#92;frac{&#92;beta_a}{&#92;mathrm{se}(&#92;beta_a)},
$$ {#eq:spredixcan}

where $&#92;sigma_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;sigma_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;beta_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (i.e., whether higher or lower predicted expression of a gene confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to find the following keywords:
 - &#92;hat{z}_{l}
 - &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)
 - &#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)}
 - $&#92;hat{&#92;sigma}_a$
 - $&#92;hat{&#92;sigma}_l$
 - $&#92;hat{&#92;beta}_a$">[FAIL] (0.66)

Failed to find the following keywords:
 - \hat{z}_{l}
 - \mathrm{se}(\hat{\gamma}_l)
 - \hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)}
 - $\hat{\sigma}_a$
 - $\hat{\sigma}_l$
 - $\hat{\beta}_a$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $z_{l}=\gamma_l / \mathrm{se}(\gamma_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
z_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\sigma_a}{\sigma_l} \frac{\beta_a}{\mathrm{se}(\beta_a)},
$$ {#eq:spredixcan}

where $\sigma_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\sigma_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\beta_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (i.e., whether higher or lower predicted expression of a gene confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to find the following keywords:
 - \hat{z}_{l}
 - \mathrm{se}(\hat{\gamma}_l)
 - \hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)}
 - $\hat{\sigma}_a$
 - $\hat{\sigma}_l$
 - $\hat{\beta}_a$</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted expression of a gene confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted expression of a gene confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves numbered equations and references to articles">Preserves numbered equations and references to articles</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.78)

PrediXcan [@doi:10.1038/ng.3367] models a trait as a linear function of a gene&#39;s expression in a single tissue, using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The association&#39;s significance is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model.

S-PrediXcan [@doi:10.1038/s41467-018-03621-1], a summary version of PrediXcan, approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

PrediXcan [@doi:10.1038/ng.3367] models a trait as a linear function of a gene's expression in a single tissue, using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The association's significance is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model.

S-PrediXcan [@doi:10.1038/s41467-018-03621-1], a summary version of PrediXcan, approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves numbered equations and references to articles">Preserves numbered equations and references to articles</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], which models the trait as a linear function of the gene&#39;s expression in a single tissue. PrediXcan uses the following univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model. In contrast, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], which models the trait as a linear function of the gene's expression in a single tissue. PrediXcan uses the following univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model. In contrast, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effects (e.g., whether higher or lower predicted expression of a gene confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effects (e.g., whether higher or lower predicted expression of a gene confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves numbered equations and references to articles">Preserves numbered equations and references to articles</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], which models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates the PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], which models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates the PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue, using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics, with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue, using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics, with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Preserves numbered equations and references to articles">Preserves numbered equations and references to articles</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a transcriptome-wide association study (TWAS) method. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a transcriptome-wide association study (TWAS) method. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.78)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where &#92;mathbf{t}_l is the gene predicted expression in tissue $l$, $&#92;beta_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;Sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where \mathbf{t}_l is the gene predicted expression in tissue $l$, $\beta_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\Sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes incorrect math references">Fixes incorrect math references</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a method that models the trait as a linear function of the gene&#39;s predicted expression in a single tissue. In PrediXcan, the univariate model is expressed as:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates the PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics, as follows:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS, with $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$ being its standard error. In both PrediXcan and S-PrediXcan, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. S-PrediXcan provides tissue-specific directions of effect, indicating whether higher or lower predicted gene expression confers increased or decreased disease risk. In our drug repurposing approach (described below), we utilized the S-PrediXcan $z$-scores to leverage this tissue-specific information.

Pass Reason: All assertions passed">[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a method that models the trait as a linear function of the gene's predicted expression in a single tissue. In PrediXcan, the univariate model is expressed as:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates the PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics, as follows:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS, with $\mathrm{se}(\hat{\beta}_a)$ being its standard error. In both PrediXcan and S-PrediXcan, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. S-PrediXcan provides tissue-specific directions of effect, indicating whether higher or lower predicted gene expression confers increased or decreased disease risk. In our drug repurposing approach (described below), we utilized the S-PrediXcan $z$-scores to leverage this tissue-specific information.

Pass Reason: All assertions passed</td>
          
          <td data-content="[FAIL] (0.41)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue, using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics, with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS, with $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$ representing its standard error. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$">[FAIL] (0.41)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue, using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics, with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS, with $\mathrm{se}(\hat{\beta}_a)$ representing its standard error. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$</td>
          
          <td data-content="[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is evaluated by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with the corresponding standard error $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are consistently estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is evaluated by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with the corresponding standard error $\mathrm{se}(\hat{\beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are consistently estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where &#92;mathbf{t}_l is the gene predicted expression in tissue $l$, $&#92;beta_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;Sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where \mathbf{t}_l is the gene predicted expression in tissue $l$, $\beta_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\Sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes incorrect math references">Fixes incorrect math references</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-level implementation of PrediXcan [@doi:10.1038/ng.3367], which models the trait as a linear function of the gene&#39;s predicted expression in a single tissue. PrediXcan uses the following univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates the PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ and $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$ are the estimated effect size and standard error of SNP $a$ from the GWAS, respectively. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-level implementation of PrediXcan [@doi:10.1038/ng.3367], which models the trait as a linear function of the gene's predicted expression in a single tissue. PrediXcan uses the following univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates the PrediXcan $z$-scores using only GWAS summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ and $\mathrm{se}(\hat{\beta}_a)$ are the estimated effect size and standard error of SNP $a$ from the GWAS, respectively. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[FAIL] (0.41)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with standard error $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$">[FAIL] (0.41)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with standard error $\mathrm{se}(\hat{\beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$</td>
          
          <td data-content="[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with the corresponding standard error $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with the corresponding standard error $\mathrm{se}(\hat{\beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where &#92;mathbf{t}_l is the gene predicted expression in tissue $l$, $&#92;beta_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;Sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where \mathbf{t}_l is the gene predicted expression in tissue $l$, $\beta_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\Sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes incorrect math references">Fixes incorrect math references</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.41)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], which models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with its standard error $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$">[FAIL] (0.41)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], which models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with its standard error $\mathrm{se}(\hat{\beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$</td>
          
          <td data-content="[FAIL] (0.41)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with standard error $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$">[FAIL] (0.41)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with standard error $\mathrm{se}(\hat{\beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$</td>
          
          <td data-content="[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with its standard error $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with its standard error $\mathrm{se}(\hat{\beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where &#92;mathbf{t}_l is the gene predicted expression in tissue $l$, $&#92;beta_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;Sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where \mathbf{t}_l is the gene predicted expression in tissue $l$, $\beta_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\Sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes incorrect math references">Fixes incorrect math references</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a method that models the trait as a linear function of the gene&#39;s expression in a single tissue. The univariate model used by PrediXcan is:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates the PrediXcan $z$-scores using only GWAS summary statistics, as follows:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS, with $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$ being its standard error. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (i.e., whether higher or lower predicted gene expression confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a method that models the trait as a linear function of the gene's expression in a single tissue. The univariate model used by PrediXcan is:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates the PrediXcan $z$-scores using only GWAS summary statistics, as follows:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS, with $\mathrm{se}(\hat{\beta}_a)$ being its standard error. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (i.e., whether higher or lower predicted gene expression confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[FAIL] (0.41)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with standard error $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$">[FAIL] (0.41)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with standard error $\mathrm{se}(\hat{\beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$</td>
          
          <td data-content="[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with the corresponding standard error $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with the corresponding standard error $\mathrm{se}(\hat{\beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where &#92;mathbf{t}_l is the gene predicted expression in tissue $l$, $&#92;beta_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;Sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where \mathbf{t}_l is the gene predicted expression in tissue $l$, $\beta_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\Sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_l$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes incorrect math references">Fixes incorrect math references</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a method that models the trait as a linear function of the gene&#39;s expression in a single tissue. In PrediXcan, the univariate model is given by:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for the gene&#39;s tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates the PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics, as follows:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with its standard error $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a method that models the trait as a linear function of the gene's expression in a single tissue. In PrediXcan, the univariate model is given by:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for the gene's tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates the PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics, as follows:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with its standard error $\mathrm{se}(\hat{\beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (indicating whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
          <td data-content="[FAIL] (0.41)

Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue, using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;epsilon_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics, with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS, with $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$ representing its standard error. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - $&#92;epsilon_l$ -&gt; $&#92;bm{&#92;epsilon}_l$">[FAIL] (0.41)

Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue, using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\epsilon_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics, with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS, with $\mathrm{se}(\hat{\beta}_a)$ representing its standard error. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - $\epsilon_l$ -> $\bm{\epsilon}_l$</td>
          
          <td data-content="[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t}_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;mathbf{t}_l$ is the predicted gene expression in tissue $l$, $&#92;gamma_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with its standard error $&#92;mathrm{se}(&#92;hat{&#92;beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed">[PASS] (0.47)

S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t}_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\mathbf{t}_l$ is the predicted gene expression in tissue $l$, $\gamma_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS with its standard error $\mathrm{se}(\hat{\beta}_a)$. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Pass Reason: All assertions passed</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes wrong latex expressions">Fixes wrong latex expressions</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a transcriptome-wide association study (TWAS) method. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l},
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (i.e., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a transcriptome-wide association study (TWAS) method. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l},
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (i.e., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l},
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only summary statistics from genome-wide association studies (GWAS) with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l},
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only summary statistics from genome-wide association studies (GWAS) with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l}
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)}
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l}
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)}
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes wrong latex expressions">Fixes wrong latex expressions</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of a gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l},
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], which models a trait as a linear function of a gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l},
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l},
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l},
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l}
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)}
$$ {#eq:spredixcan}

Here, $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l}
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)}
$$ {#eq:spredixcan}

Here, $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes wrong latex expressions">Fixes wrong latex expressions</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a transcriptome-wide association study (TWAS) method. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l},
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (i.e., whether a higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a transcriptome-wide association study (TWAS) method. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l},
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (i.e., whether a higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l},
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l},
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. While PrediXcan requires individual-level data to fit this model, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l}
$$ (Equation 1)

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)}
$$ (Equation 2)

Here, $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l}
$$ (Equation 1)

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)}
$$ (Equation 2)

Here, $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes wrong latex expressions">Fixes wrong latex expressions</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a transcriptome-wide association study (TWAS) method. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l},
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (i.e., whether higher or lower predicted expression of a gene confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary-based version of PrediXcan [@doi:10.1038/ng.3367], a transcriptome-wide association study (TWAS) method. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l},
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (i.e., whether higher or lower predicted expression of a gene confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l},
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l},
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l}
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)}
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l}
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)}
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
        </tr>
        
        <tr>
          
          <td data-content="S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression on a single tissue using the univariate model

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l,
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
&#92;hat{z}_{l} &#92;appro &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of SNP $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).">S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is the summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression on a single tissue using the univariate model

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l,
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan needs individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only GWAS summary statistics with the expression

$$
\hat{z}_{l} \appro \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of SNP $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these TWAS methods, the genotype variances and covariances are always estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (for instance, whether a higher or lower predicted expression of a gene confers more or less disease risk), we used the $z$-scores in our drug repurposing approach (described below).</td>
          
          <td data-content="genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits">genetic studies, functional genomics, gene co-expression, therapeutic targets, drug repurposing, clustering of complex traits</td>
          
          <td data-content="Fixes wrong latex expressions">Fixes wrong latex expressions</td>
          
          <td data-content="Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms">Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367], a method that models the trait as a linear function of the gene&#39;s expression in a single tissue. PrediXcan uses the following univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l},
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model. In contrast, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the following expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (i.e., whether a higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367], a method that models the trait as a linear function of the gene's expression in a single tissue. PrediXcan uses the following univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l},
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model. In contrast, S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the following expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted expression of a gene in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific direction of effects (i.e., whether a higher or lower predicted expression of a gene confers more or less disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l},
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ are the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)},
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l},
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ are the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)},
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
          <td data-content="[FAIL] (0.34)

Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene&#39;s expression in a single tissue using the univariate model:

$$
&#92;mathbf{y} = &#92;mathbf{t_l &#92;gamma_l + &#92;bm{&#92;epsilon}_l}
$$ {#eq:predixcan}

where $&#92;hat{&#92;gamma}_l$ is the estimated effect size or regression coefficient, and $&#92;bm{&#92;epsilon}_l$ represents the error terms with variance $&#92;sigma_{&#92;epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $&#92;hat{z}_{l}=&#92;hat{&#92;gamma}_l / &#92;mathrm{se}(&#92;hat{&#92;gamma}_l)$ for a gene&#39;s tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
&#92;hat{z}_{l} &#92;approx &#92;sum_{a &#92;in model_{l}} w_a^l &#92;frac{&#92;hat{&#92;sigma}_a}{&#92;hat{&#92;sigma}_l} &#92;frac{&#92;hat{&#92;beta}_a}{&#92;mathrm{se}(&#92;hat{&#92;beta}_a)}
$$ {#eq:spredixcan}

where $&#92;hat{&#92;sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $&#92;hat{&#92;sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $&#92;hat{&#92;beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - &#92;mathbf{t_l  -&gt; &#92;mathbf{t}_l">[FAIL] (0.34)

Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l 
---
S-PrediXcan [@doi:10.1038/s41467-018-03621-1] is a summary version of PrediXcan [@doi:10.1038/ng.3367]. PrediXcan models the trait as a linear function of the gene's expression in a single tissue using the univariate model:

$$
\mathbf{y} = \mathbf{t_l \gamma_l + \bm{\epsilon}_l}
$$ {#eq:predixcan}

where $\hat{\gamma}_l$ is the estimated effect size or regression coefficient, and $\bm{\epsilon}_l$ represents the error terms with variance $\sigma_{\epsilon}^{2}$. The significance of the association is assessed by computing the $z$-score $\hat{z}_{l}=\hat{\gamma}_l / \mathrm{se}(\hat{\gamma}_l)$ for a gene's tissue model $l$. PrediXcan requires individual-level data to fit this model, whereas S-PrediXcan approximates PrediXcan $z$-scores using only genome-wide association study (GWAS) summary statistics with the expression:

$$
\hat{z}_{l} \approx \sum_{a \in model_{l}} w_a^l \frac{\hat{\sigma}_a}{\hat{\sigma}_l} \frac{\hat{\beta}_a}{\mathrm{se}(\hat{\beta}_a)}
$$ {#eq:spredixcan}

where $\hat{\sigma}_a$ is the variance of single nucleotide polymorphism (SNP) $a$, $\hat{\sigma}_l$ is the variance of the predicted gene expression in tissue $l$, and $\hat{\beta}_a$ is the estimated effect size of SNP $a$ from the GWAS. In these transcriptome-wide association study (TWAS) methods, the genotype variances and covariances are estimated using the Genotype-Tissue Expression project (GTEx v8) [@doi:10.1126/science.aaz1776] as the reference panel. Since S-PrediXcan provides tissue-specific directions of effect (e.g., whether higher or lower predicted gene expression confers increased or decreased disease risk), we utilized the $z$-scores in our drug repurposing approach (described below).

Fail Reason: Failed to fix the following expressions:
 - \mathbf{t_l  -> \mathbf{t}_l</td>
          
        </tr>
        
      </tbody>
    </table>
  </body>
</html>
